src/services/TemplatingService.ts

```diff
@@ -13,79 +16,85 @@ export default class TemplatingService {
   // endregion
 
   // region private static methods
+  private static initializeMessageTemplates(platformId: string, templateId: string): Template[] {
+    return [MAIL_MESSAGE, PUSH_MESSAGE].map((type) => {
+      const messageTemplate = Utils.loadTemplate(platformId, templateId, type);
+      TemplatingService.parseMessageTemplate(messageTemplate);
+      return messageTemplate;
+    });
+  }
 
-  private static insertMessagePayload(template: any, payload: any, platformId: string): any {
-    const config = Utils.getPlatformConfig(platformId);
-    return {
-      from: config.mail.defaults.from,
-      data: template.data,
-    };
+  private static parseMessageTemplate(template: Template) {
```

Comment:
The templates for push messages are nested. This is currently not supported, right?

The PUSH template could look like this:
```
apns:
    payload:
        aps:
            title: "{{ message.title }}"
            body: "{{ message.body }}"
```

Reply:
Done